src/service/iot_config.proto

```diff
@@ -378,19 +386,36 @@ service route {
   rpc get(route_get_req_v1) returns (route_v1);
   // Create Route for an Org (auth delegate_keys/owner/admin)
   rpc create(route_create_req_v1) returns (route_v1);
+
```

Comment:
Now that we've litted `euis` and `devaddrs` up a level, should we clarify `_route` on these endpoints?

Reply:
it's probably fine since it's all off the `route` service, but an rpc definition of `iot_config.route/create_route` doesn't read too bad to me, either.